pkg/cachemanager/fakecfdataclient.go

```diff
@@ -11,51 +11,50 @@ WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
 See the License for the specific language governing permissions and
 limitations under the License.
 */
-package fakes
+package cachemanager
 
 import (
 	"context"
 	"fmt"
 	gosync "sync"
 
 	constraintTypes "github.com/open-policy-agent/frameworks/constraint/pkg/types"
-	"github.com/open-policy-agent/gatekeeper/v3/pkg/syncutil"
 	"github.com/open-policy-agent/gatekeeper/v3/pkg/target"
 	"k8s.io/apimachinery/pkg/runtime/schema"
 	"sigs.k8s.io/controller-runtime/pkg/client"
 )
 
-type OpaKey struct {
+type CfDataKey struct {
 	Gvk schema.GroupVersionKind
 	Key string
 }
 
-// FakeOpa is an OpaDataClient for testing.
-type FakeOpa struct {
+// FakeCfClient is an CfDataClient for testing.
+type FakeCfClient struct {
 	mu           gosync.Mutex
-	data         map[OpaKey]interface{}
+	data         map[CfDataKey]interface{}
 	needsToError bool
 }
 
-var _ syncutil.OpaDataClient = &FakeOpa{}
+var _ CFDataClient = &FakeCfClient{}
 
-// keyFor returns an opaKey for the provided resource.
+// keyFor returns a cfDataKey for the provided resource.
 // Returns error if the resource is not a runtime.Object w/ metadata.
-func (f *FakeOpa) keyFor(obj interface{}) (OpaKey, error) {
+func (f *FakeCfClient) keyFor(obj interface{}) (CfDataKey, error) {
```

Comment:
You could make this a non-receiver method (it doesn't use the `f`) and then use it in your test to create all your CfDataKey variables.  That way you don't have to maintain a bunch of magic strings.

Reply:
That's a good call out! I took out the receiver and started to use it in some of the tests to generate the keys on the fly.